src/data/restore.ts

```diff
@@ -1,45 +1,49 @@
+import oc from "open-color";
+import { getDefaultAppState } from "../appState";
+import {
+  DEFAULT_FONT_FAMILY,
+  DEFAULT_TEXT_ALIGN,
+  DEFAULT_VERTICAL_ALIGN,
+  FONT_FAMILY,
+  PRECEDING_ELEMENT_KEY,
+  ROUNDNESS,
+} from "../constants";
+import {
+  getNonDeletedElements,
+  getNormalizedDimensions,
+  isInvisiblySmallElement,
+  refreshTextDimensions,
+} from "../element";
+import { LinearElementEditor } from "../element/linearElementEditor";
+import { bumpVersion } from "../element/mutateElement";
+import {
+  detectLineHeight,
+  getDefaultLineHeight,
+  measureBaseline,
+} from "../element/textElement";
+import { isTextElement, isUsingAdaptiveRadius } from "../element/typeChecks";
```

Comment:
Same thing here about the automatic sorting: https://github.com/excalidraw/excalidraw/pull/6479/files#r1169914272